subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -341,6 +341,16 @@ static bool schedule_send(struct bt_mesh_ext_adv *adv)
 	     adv->tags & BT_MESH_ADV_TAG_BIT_FRIEND) ||
 	    (CONFIG_BT_MESH_RELAY_ADV_SETS > 0 && adv->tags & BT_MESH_ADV_TAG_BIT_RELAY)) {
 		k_work_reschedule(&adv->work, K_NO_WAIT);
+	} else if ((IS_ENABLED(CONFIG_BT_MESH_PB_ADV)) && adv->tags & BT_MESH_ADV_TAG_BIT_PROV) {
```

Comment:
This won't work. It should be this tag https://github.com/zephyrproject-rtos/zephyr/blob/a15ef7927b23088bf016966daa8879fa4a128a78/subsys/bluetooth/mesh/adv.h#L51